abseil/abseil-cpp#2160 -- see patches/patch-abseil2160-absl_base_internal_raw__logging.cc for the why.

diff --git a/absl/base/internal/raw_logging.cc b/absl/base/internal/raw_logging.cc
--- a/absl/base/internal/raw_logging.cc
+++ b/absl/base/internal/raw_logging.cc
@@ -44,6 +44,7 @@
 #if defined(__linux__) || defined(__APPLE__) || defined(__FreeBSD__) || \
     defined(__hexagon__) || defined(__Fuchsia__) ||                     \
     defined(__native_client__) || defined(__OpenBSD__) ||               \
+    defined(__NetBSD__) || defined(__DragonFly__) ||                    \
     defined(__EMSCRIPTEN__) || defined(__ASYLO__)
 
 #include <unistd.h>
